File: FitToFit/FitToFit.Services/Database/RezervacijaStavke.cs
using System;
using System.Collections.Generic;

namespace FitToFit.Services.Database;

public partial class RezervacijaStavke
{
    public int RezervacijaStavkeId { get; set; }

    public int RezervacijaId { get; set; }

    public int TerminId { get; set; }

    public virtual Rezervacije Rezervacija { get; set; } = null!;

    public virtual Termini Termin { get; set; } = null!;
}

File: FitToFit/UI/fittofit_admin/lib/models/rezervacijaStavke.dart
// ignore_for_file: file_names

import 'package:fittofit_admin/models/termini.dart';
import 'package:json_annotation/json_annotation.dart';

part 'rezervacijaStavke.g.dart';

@JsonSerializable()
class RezervacijaStavke {
  int rezervacijaStavkeId;
  int rezervacijaId;
  int terminId;
  Termini? termin;

  RezervacijaStavke(
      {required this.rezervacijaStavkeId,
      required this.rezervacijaId,
      required this.terminId,
      this.termin});

  factory RezervacijaStavke.fromJson(Map<String, dynamic> json) =>
      _$RezervacijaStavkeFromJson(json);

  Map<String, dynamic> toJson() => _$RezervacijaStavkeToJson(this);
}
